src/uglify/Runner.js

```diff
@@ -16,7 +16,10 @@ export default class Runner {
   constructor(options = {}) {
     const { cache, parallel } = options;
     this.cacheDir = cache === true ? findCacheDir({ name: 'uglifyjs-webpack-plugin' }) : cache;
-    this.maxConcurrentWorkers = parallel === true ? os.cpus().length - 1 : Math.min(Number(parallel) || 0, os.cpus().length - 1);
+    // https://github.com/nodejs/node/issues/19022
+    // in some cases cpus() returns undefined
+    const cpusLength = (os.cpus() ? os.cpus().length : 1);
```

Comment:
Let's rewrite to `const cpusLength = (os.cpus() || { length: 1 }).length` as said https://github.com/nodejs/node/issues/19022#issuecomment-374275868

Reply:
Sorry, I don't know how this is done. Can you help out?

Reply:
@vielhuber change code in you branch, commit changes and push with force. Sorry i don't have time to learn, better read book about `git`. You can create new PR if you don't know how do this.

Reply:
Done.